### PR TITLE
Add image, replace plus button with secondary button

### LINKED ISSUE #3478: Add image button should be a regular button
Right now the Images page has a big `+` sign in a blue circle in the top right corner to "Add an image".

This is accessibility issues, so would need at least to be labeled. But it may be simpler (and easier to understand for the casual user) if we used a regular button with text.

Implement a secondary call-to-action button called "Add Image" (no icon).

### PR DESCRIPTION
**Pull request by torchiaf**
Fixes #3478

Signed-off-by: Francesco Torchia <francesco.torchia@suse.com>

### CODE REVIEW

File: pkg/rancher-desktop/components/ImagesButtonAdd.vue [hunk 1]
```diff
@@ -32,21 +24,15 @@ export default Vue.extend({
...
-  justify-content:center;
-}
 
-.icon {
-  width: 0.75rem;
-  height: 0.75rem;
-}
+  .btn {
+    margin-top: -3px;
+  }
```
Comment (rak-phillip) [inline pkg/rancher-desktop/components/ImagesButtonAdd.vue:30]: What's the benefit to the negative margin? I don't think that we would want this for the button.

**Review by rak-phillip — CHANGES_REQUESTED** [review]
Just have one question. Looks good otherwise.

File: pkg/rancher-desktop/components/ImagesButtonAdd.vue (hunk 1 above)
Comment (ericpromislow) [reply pkg/rancher-desktop/components/ImagesButtonAdd.vue:30]: +1 on Phillip's comment -- I don't see the difference between this code and leaving out the margin-top directive. For those of us who aren't CSS experts, less (CSS code) is more.

Comment (torchiaf) [reply pkg/rancher-desktop/components/ImagesButtonAdd.vue:30]: It was supposed to put the button at the same distance from top and bottom lines of the header:

`margin-top -3px` :
image 
image


`margin-top 0` :
image

Let's remove the margin directive, we can always fix the layout later.

**Review by rak-phillip — APPROVED** [review]
LGTM

**Pull request merged**